syntax = "proto3";

import "Common.proto";

package proto;

message C2S_EnterRoom
{
    int64   id = 1;
    string  password = 2;
}

message C2S_Chat
{
    int64   id = 1;
    string  message = 2;
}